### PR TITLE
Install kepler dashboard

### LINKED ISSUE #16: Install Grafana with Flux
## Description

Install Grafana with Flux using its Helm chart
 
## Issue

Parent: https://github.com/cncf-tags/green-reviews-tooling/issues/5

### PR DESCRIPTION
**Pull request by dipankardas011**
## Description

it provides a yaml for the kepler grafana dashboard and docs/ for documentation of steps taken

## Tasks
- [ ] Closes #16 

> **Note**
> it uses kube-prometheus-stack

### CODE REVIEW

**Comment by dipankardas011** [comment]
Updated the kepler flux manifest
@rossf7 @AntonioDiTuri

File: docs/full-kepler.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Full Kepler install with dashboard
```
[the remark this answers is not in the document]
Comment (dipankardas011) [reply docs/full-kepler.md:1]: Sure

File: clusters/kepler-grafana.yaml [hunk 2]
```diff
@@ -0,0 +1,984 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kepler-dashboard
```
[the remark this answers is not in the document]
Comment (dipankardas011) [reply clusters/kepler-grafana.yaml:4]: Okay it doesn't matter

**Review by rossf7 — APPROVED** [review]
lgtm

**Pull request merged**